steward/src/cork.rs

```diff
@@ -79,6 +87,12 @@ impl steward::contract_call_server::ContractCall for CorkHandler {
             id: id_hash(height, &cellar_id, encoded_call),
         }))
     }
+
+    async fn status(&self, _: Request<StatusRequest>) -> Result<Response<StatusResponse>, Status> {
+        Ok(Response::new(StatusResponse {
+            version: STEWARD_VERSION.to_string(),
```

Comment:
Can we store the static value as a `String` so we don't have to convert it every time?

Reply:
With the way lazy_static works we'd still need the `.to_string()` call

Reply:
Oh you mean store it separately from the static, I can do that

Reply:
We would still have to clone it each time to avoid moving out of the stored location, which is probably a similar cost right?